src/store/Settings.js

```diff
@@ -125,7 +125,7 @@ const actions = {
       totalAmountInterval: 0
     }
     return irohaUtil.setAccountDetail(privateKeys, getters.irohaQuorum, {
-      accountId: getters.accountId,
+      accountId: state.accountId,
```

Comment:
Need to check, I guess this `state.accountId` can be `undefined`. Same for line 144